.generated_docs

```diff
@@ -23,12 +23,14 @@ docs/man/man1/kubectl-convert.1
 docs/man/man1/kubectl-create.1
 docs/man/man1/kubectl-delete.1
 docs/man/man1/kubectl-describe.1
+docs/man/man1/kubectl-drain.1
 docs/man/man1/kubectl-edit.1
 docs/man/man1/kubectl-exec.1
 docs/man/man1/kubectl-explain.1
 docs/man/man1/kubectl-expose.1
 docs/man/man1/kubectl-get.1
 docs/man/man1/kubectl-label.1
+docs/man/man1/kubectl-lame.1
```

Comment:
rotate in/out and enable/disable have the problem @bgrant0607 mentioned, namely that they connote the wrong thing 

make-unschedulable and make-schedulable, while annoyingly wordy, are probably the most unambiguous alternatives to lame/unlame